build/index.js

```diff
@@ -159,11 +160,7 @@ function executeProcess(executable, args, env = process.env) {
 
 const npm = async (...args) => {
     const start = new Date();
-    await executeProcess(
-        "npm",
-        ["--no-update-notifier", "--no-fund", ...args],
-        { ...process.env, FORCE_COLOR: 1 }
-    );
+    await executeProcess("npm", ["--no-update-notifier", "--no-fund", ...args]);
```

Comment:
@JJ Did you mean to remove the glorious `FORCE_COLOR`? :D

Reply:
Not me! Put it back!